packages/select/README.md

```diff
@@ -1,10 +1,16 @@
 # [`@fremtind/jkl-select`](https://fremtind.github.io/jokul/components/select/)
 
-Vi bruker nedtrekkslister for å gi brukerne mulighet til å velge et alternativ i en liste med valg. Som standard er ingen av alternativene valgt på forhånd. Vi bruker gjerne nedtrekkslister når det er mer enn tre valg. Hvis det er færre valg, bruker vi radioknapper.
+# Nedtrekksliste
+Nedtrekkslister lar brukeren velge et alternativ i en liste med valg. De brukes når det er for mange valg for radioknapper. Ledeteksten beskriver hva som skal velges.
 
-Nedtrekkslister skal alltid ha en ledetekst som tydelig forklarer hva listen inneholder valg for. Vi kan også legge til en hjelpetekst under, hvis vi trenger å forklare valgene nærmere.
+For nedtrekkslister med veldig mange valg kan feltet også ha en innebygd søke / filtreringsfunksjon.
```

Comment:
Per nå er ikke dette noe vi har støtte for er det? Mulig jeg bare ikke har fått det med meg

Reply:
Finnes ikke per nå, nei. Det støttes å skrive inn det man vil velge, men det eneste som skjer er at fokus flyttes til nærmeste treff i listen. Det er nok et mønster vi bør implementere!